configure.ac

```diff
@@ -218,6 +218,23 @@ AS_IF([test "x$enable_exp_cq" == xyes],
 	])
 )
 
+# Device Memory
```

Comment:
this is shorter
AC_CHECK_DECL([IBV_EXP_DEVICE_ATTR_MAX_DM_SIZE,
		[AC_DEFINE(HAVE_IBV_DM, 1, [Define to 1 if IBV_EXP_DEVICE_ATTR_MAX_DM_SIZE is defined])],
		[],
		[[#include <infiniband/verbs_exp.h>]])

Reply:
+1